pennylane_cirq/simulator_device.py

```diff
@@ -42,6 +43,15 @@
 from .cirq_device import CirqDevice
 
 
+@functools.lru_cache()
+def z_eigs(n):
+    """Return the eigenvalues of an n-fold tensor product of Pauli Z operators."""
```

Comment:
This function docstring is better than the one in the qiskit plugin, might copy it back :laughing: